products/sce/imds/Container-status-endpoint.mdx

```diff
@@ -3,40 +3,24 @@ title: 'Using IMDS to get the status of a replica'
 ---
 
 We can use the IMDS to retrieve the current status of the replica from inside the workload. This can be achieved using
-either the SDK, or the JSON request endpoint. In this example, we'll create a Python script using the SDK to check
-whether the container is ready, and start a bash script when it is.
+either the IMDS SDK, or the JSON request endpoint. In this example, we'll create a Python script using the IMDS SDK to
+check whether the container is ready, and start a bash script when it is.
 
 ## Using the SDK to retrieve the status
 
-We'll start by using the SDK to retrieve the statuses of the currently running container.
+We'll start by using the IMDS SDK to retrieve the statuses of the currently running container.
 
+<CodeGroup>
 ```python Python
 from salad_cloud_imds_sdk import SaladCloudImdsSdk, Environment
 
-sdk = SaladCloudImdsSdk(
-    base_url=Environment.DEFAULT.value,
-    timeout=10000
-)
+sdk = SaladCloudImdsSdk( base_url=Environment.DEFAULT.value, timeout=10000 )
 
 result = sdk.metadata.get_container_status()
 
 print(result)
-```
-
-Now, if we run this on a container that is currently `started`, but not yet `ready`, we'll see this:
-
-```
-{
-  "ready": false,
-  "started": true
-}
-```
 
-### Using a different language?
-
-You can find examples in other languages of the SDK usage here.
-
-<CodeGroup>
+````
```

Comment:
Do you mean to have 4 backtick's (`) here? Just below you correct this and I know 3 backticks is a pretty common notation...

Reply:
The auto-formatting done by the config file in the repo seems to add these, every time I save the file it re-adds it. Doesn't seem to cause any issues, but not sure why it does it in some cases but not all.